### ISSUE TITLE
🐛 Unexpected error in `expr_ext.rs`

### ISSUE THREAD

**Issue by arendjr** [biomejs/biome#450]
### Environment information

```block
Just received the following error from Biome in my console:


This is a bug in Biome, not an error in your code, and we would appreciate it if you could report it to https://github.com/biomejs/biome/issues/ along with the following information to help us fixing the issue:

Source Location: crates/biome_js_syntax/src/expr_ext.rs:633:26
Thread Name: tokio-runtime-worker
Message: begin <= end (1 <= 0) when slicing `/`
```

### What happened?

I was simply working in VS Code with the extension enabled and typed `//` to start a new comment.

### Expected result

It should not error.

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by Conaclos** [comment]
As a temporary fix, I recommend disabling [noControlCharactersInRegex](https://biomejs.dev/linter/rules/no-control-characters-in-regex).
